frontend/src/metabase-lib/comparison.ts

```diff
@@ -14,6 +15,17 @@ export function areLegacyQueriesEqual(
   return ML.query_EQ_(query1, query2, fieldIds);
 }
 
+export function isSameColumn(
+  query: Query,
+  stageIndex: number,
+  column1: ColumnMetadata,
+  column2: ColumnMetadata,
+) {
+  const name1 = displayInfo(query, stageIndex, column1).longDisplayName;
+  const name2 = displayInfo(query, stageIndex, column2).longDisplayName;
+  return name1 === name2;
+}
+
```

Comment:
This is a placeholder that works good enough for demo purposes, but it should better be a CLJS method